dags/utils/config_parser.py

```diff
@@ -29,13 +29,25 @@ def __init__(self, path=None):
     def _get_country(self, country) -> FallbackDict:
         return FallbackDict(self.raw_config, self.country_config[self._country_lower(country)])
 
-    def name_output_file_generic(self, geo_extent, category, theme, geometry, scale, source, permission,
-                                 free_text=None):
+    def _get_name_output_file_generic(self, country: str, filename_field: FallbackDict) -> str:
+        geo_extent = self.get_iso3(country).lower()
+        file_name = self._name_output_file_generic(geo_extent, filename_field['category'], filename_field['theme'],
```

Comment:
We prefer naming parameters when calling to avoid mistakes

```
file_name = self._name_output_file_generic(geo_extent=geo_extent, category=filename_field['category'],
                                           theme=filename_field['theme'], geometry=filename_field['geometry'],
                                           scale=filename_field['scale'], source=filename_field['source'],
                                           suffix=filename_field['suffix'])
```

Reply:
Sure, will do! Do you have any advice on how to make this part a bit nicer? I wanted to just pass `*filename_field` but it doesn't work since it's not actually a dictionary (and maybe that anyway goes against your parameter naming convention).

Reply:
For the dictionary unpacking you would need `**`, but I guess you meant that. I would keep it like this as long as the parameter list is probably not going to grow a lot. This is clear and straightforward and easy to understand.

Reply:
Okay thanks!